scripts/get-unowned-packages/index.js

```diff
@@ -19,10 +19,18 @@ const getPackagesWithReadWriteAccess = async user => {
   }, {})
 }
 
-module.exports = function getUnownedPackages({
+module.exports = async function getUnownedPackages({
   rootPath = path.join(__dirname, `../..`),
   user,
 } = {}) {
+  // infer user from npm whoami
+  // set registry because yarn run hijacks registry
+  if (!user) {
```

Comment:
I moved this up as we only need to do this check once and not for every package.